frontend/src/component/playground/Playground/PlaygroundResultsTable/FeatureResultInfoPopoverCell/FeatureDetails/FeatureDetails.test.tsx

```diff
@@ -10,7 +11,7 @@ const testCases = [
             hasUnsatisfiedDependency: true,
             isEnabledInCurrentEnvironment: false,
         } as PlaygroundFeatureSchema,
-        expectedText1: 'This feature flag is False in development because',
+        expectedText1: /This feature flag is False in development because/,
```

Comment:
The exact HTML structure has changed a little bit in the new component (from `<p><span><span><span></p>` to `<p>text<span>text</p>`), so the old matcher didn't work correctly. But making it a regex seems harmless enough and shows that it can still find the text.